### PR TITLE
Add getAccountBalance() API.

### LINKED ISSUE #2189: [Feature] API call for getting account balance
### Verified feature request does not already exist?

- [X] I have searched and found no existing issue

### 💻

- [ ] Would you like to implement this feature?

### Pitch: what problem are you trying to solve?

Hi there, I'm trying to implement my own tools to interact with the Actual, but I didn't find any API calls that could simply give me the total balance of an account. Although I probably could just get all transactions with a super early starting date and sum them up, it's a little bit more tedious than getting the value in one go. 
Just curious, is this intentional by design? Why the `Account` object can't simply include a "balance" field? Or am I missing anything?
Thanks.

### Describe your ideal solution to this problem

Add `balance` field to `Account`, which has the current balance value of that account.

### Teaching and learning

_No response_

### PR DESCRIPTION
**Pull request by psybers**
Closes #2189 by adding a `getAccountBalance()` API function that takes the account ID as first argument, and an optional date cutoff (for finding balances on previous days).

### CODE REVIEW

**Review by joel-jeremy — DISMISSED** [review]

File: packages/loot-core/src/types/api-handlers.d.ts [hunk 1]
```diff
@@ -114,6 +114,8 @@ export interface ApiHandlers {
 
   'api/account-delete': (arg: { id }) => Promise<unknown>;
 
+  'api/account-balance': (arg: { id; cutoff? }) => Promise<number>;
```
Comment (MatissJanis) [inline packages/loot-core/src/types/api-handlers.d.ts:117]: **💬 suggestion:** ‏
```suggestion
  'api/account-balance': (arg: { id: string; cutoff?: Date }) => Promise<number>;
```

**Review by MatissJanis — APPROVED** [review]
Great! Thanks for working on all these API improvements!

**Comment by youngcw** [comment]
Did this get added to the docs?  I don't see a PR for it

**Comment by psybers** [comment]
@youngcw No, I think the docs were not updated yet.

**Comment by psybers** [comment]
See: https://github.com/actualbudget/docs/pull/403

**Pull request merged**